src/controllers/baseHandler.ts

```diff
@@ -36,14 +36,12 @@ export async function baseHandler(
     }
     case getCommandName(MENTION_EACH): {
       const data = message.data?.options as Array<messageRequestDataOptions>;
+      const transformedArgument = {
+        roleToBeTaggedObj: data[0],
+        displayMessageObj: data[1] ?? {},
```

Comment:
if i am not mistaken data[0]is the roles option in command and data[1] is the message we want to send, Dould we maybe mention it somewhere or rename the variables

Reply:
hey ritik added comments for this.